bokeh/command/subcommands/serve.py

```diff
@@ -311,6 +311,20 @@
     DEFAULT_LOG_FORMAT=DEFAULT_LOG_FORMAT
 )
 
+def _fixup_deprecated_host_args(args):
+    if args.host is not None and len(args.host) > 0:
+        if args.allow_websocket_origin is None:
+            args.allow_websocket_origin = []
+        args.allow_websocket_origin += args.host
+        args.allow_websocket_origin = list(set(args.allow_websocket_origin))
+        warnings.warn(
+            "The --host parameter is deprecated because it is no longer needed. "
+            "It will be removed and trigger an error in a future release. "
+            "Values set now will be copied to --allow-websocket-origin. "
+            "Depending on your use case, you may need to set current --host "
+            "values for 'allow_websocket_origin' instead."
+        )
```

Comment:
This is why I am thinking of making it an `info` instead of a warning. I would have to be convinced not ro deprecate it. It is the single biggest source of confusion and difficulty and questions and problems I have to field about serving bokeh apps.